src/gdb.ts

```diff
@@ -422,8 +424,13 @@ export class GDBDebugSession extends DebugSession {
                         }
                     };
 
-                    if (this.args.runToMain) {
-                        this.miDebugger.sendCommand('break-insert -t --function main').then(() => {
+                    if (this.args.runToEntryPoint) {
+                        let entryPointName = this.args.entryPointName || DEFAULT_ENTRY_POINT_NAME;
```

Comment:
When entryPointName is not specified, I’m not sure whether is better to silently set the breakpoint to the default value (e.g. by setting entryPointName default value in the config provider) or to let the user know that the entry point name can be defined and fallback to the default value (current implementation)

Reply:
I don't think I'd bother with the message myself; that way it can largely function as a direct replacement for `runToMain` without having, just with the extra configurability for if they want a different name.

Myself I also think it's probably a bit cleaner to just set the `entryPointName` value to `main` in the config provider if it isn't already set.

Reply:
Updated to set default value of `entryPointName` to `main` in config provider